api_app/models/schemas/resource.py

```diff
@@ -5,11 +5,15 @@
 class ResourcePatch(BaseModel):
     isEnabled: Optional[bool]
     properties: Optional[dict]
+    templateVersion: Optional[str]
+    forceVersionUpdate: Optional[bool]
```

Comment:
Will this param be persisted to the db?
Have you considered providing it as a query param rather than in the payload?

Reply:
Do we have other places where we include a command param in the payload and don't persist it?
Is it easy to make this into a query param?